README.md

```diff
@@ -1,24 +1,49 @@
 [![unstable](http://badges.github.io/stability-badges/dist/unstable.svg)](http://github.com/badges/stability-badges)
 
-This project explores common infrastructure and coding standards for open source Google Cloud Platform projects.
+This project explores tools and practices 
+to provide consistency among Google Cloud Java libraries. The consistency
+minimizes the risk of dependency conflicts caused by the libraries.
 
-[Code Hygiene Dashboard](https://storage.googleapis.com/cloud-opensource-java-dashboard/dashboard/target/dashboard/dashboard.html) (runs daily; work in progress)
+# Tools and Practices
 
-To generate the dashboard from the root directory run:
+## Google Cloud Platform Java Dependency Dashboard
 
-```
-$ mvn clean install
-$ mvn exec:java -Dexec.arguments="-f boms/cloud-oss-bom/pom.xml"
-```
+[Google Cloud Platform Java Dependency Dashboard](
+https://storage.googleapis.com/cloud-opensource-java-dashboard/dashboard/target/dashboard/dashboard.html)
+(runs daily; work in progress) shows multiple checks on the consistency among
+Google Cloud Java libraries. For manually generating the dashboard, see
+[its README](./dashboard/README.md).
+
+## Java Library Best Practices
+
+[The Java Library Best Practices](./library-best-practices) are a set of rules
+that we believe will minimize problems for consumers of interconnected Java
+libraries.
+
+## Linkage Checker Enforcer Rule
+
+[Linkage Checker Enforcer Rule](./enforcer-rules) verifies that the transitive
+dependency tree of a Maven project does not have any [linkage errors](
+./library-best-practices/glossary.md#types-of-conflicts-and-compatibility).
+
+## Google Library BOM
+
+[Google Library BOM](boms/cloud-oss-bom) is a Bill-of-Materials (BOM) that
+lists of Google Cloud Java libraries with their versions.
```

Comment:
delete "of"
better yet, "provides consistent versions of Google Cloud Java libraries that work together without linkage errors"